pkg/render/render.go

```diff
@@ -41,6 +41,7 @@ var (
 )
 
 type Component interface {
+	ResolveImages(is *operator.ImageSet) error
```

Comment:
This should have a comment explaining what the expected behavior of this function is.